sources/Application/Model/Project.h

```diff
@@ -10,7 +10,7 @@
 #include "Foundation/Variables/VariableContainer.h"
 #include "Song.h"
 
-#define PROJECT_NUMBER "2.1-BETA1"
+#define PROJECT_NUMBER "2.1-JumpSection"
```

Comment:
this is fine to change in your local test builds but please dont commit this into the PR branch

Reply:
Should I remove the commit changing this completely from the commit-history in the PR branch? Or is reverting it sufficient? (e.g. if PRs are being squashed anyways)

Reply:
just reverting or a new commit putting the old line back is fine since yes we do squash merge PRs so the commits in a PR branch arent kept anyways